vmdb/lib/workers/worker_base.rb

```diff
@@ -88,16 +90,14 @@ def self.wait_for_worker_monitor=(val)
 
   def my_monitor_started?
     return @monitor_started unless @monitor_started.nil?
-    server = MiqServer.my_server(true)
     return false if     server.nil?
-    return false unless server.started?
+    return false unless server.reload.started?
```

Comment:
Originally, the local var `server` was being loaded directly from the DB.  By introducing the `attr_reader`, I wanted to not hide the reader method with the local var.  So, calling `#reload` replicates the original logic.  I'm not entirely sure it's necessary to reload the server record, but looking at the `started?` attribute implies that the server record needs to be up to date (i.e., make sure the server didn't stop since the last time the `@server` ivar was init'd.